packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio-group/tests/gux-form-field-radio-group.e2e.ts

```diff
@@ -0,0 +1,145 @@
+import { E2EPage, newE2EPage } from '@stencil/core/testing';
+import {
+  newSparkE2EPage,
+  a11yCheck
+} from '../../../../../../test/e2eTestUtils';
+
+const axeExclusions = [];
+
+async function newNonrandomE2EPage({
+  html
+}: {
+  html: string;
+}): Promise<E2EPage> {
+  const page = await newE2EPage();
+
+  await page.evaluateOnNewDocument(() => {
+    Math.random = () => 0.5;
+  });
```

Comment:
We need to add this so that the randomized ids that are added for accessibility reasons do not fail the snapshot tests each time. For example, if I comment out this line I get failing snapshot tests like this: 
![image](https://github.com/MyPureCloud/genesys-spark/assets/82100934/f0849042-bec6-4c63-82af-b7798f0da014)